monailabel/tasks/train/basic_train.py

```diff
@@ -157,7 +157,7 @@ def __init__(
         self._config = {
             "name": "train_01",
             "pretrained": True,
-            "device": "cuda",
+            "device": device_displayname_list()[0],
```

Comment:
Hi @japarada , now the training works well. One question, why here to use the first item of `device_displayname_list()`, this will make the config option from drop-down options to the text editor. Shall we just use `device_displayname_list()`?